spec/unit/recipes/add_to_list_missing_file_spec.rb

```diff
@@ -20,7 +20,7 @@
 require 'chefspec'
 require 'chefspec/berkshelf'
 
-describe 'test::add_to_list_missing_file' do
+describe 'test::add_to_list_missing_file_rspec' do
```

Comment:
rspec?

Reply:
Working on the names.  Those files get used by the rspec tests.  I'll combine the other files.

Reply:
In that case shouldn't they be "inspec"? 

Having the name at the end seems superfluous .

Reply:
I'm going to move them over to the spec directory. They get used by running rspec.

Reply:
fixed